Changes

```diff
@@ -378,7 +378,11 @@ Next version (4.05.0):
   (Alain Frisch, report by Anil Madhavapeddy)
 
 - PR#7443, GPR#990: spurious unused open warning with local open in patterns
-  (Florian Angeletti, report by Gabriel Scherer)
+  (Florian Angeletti, report by Gabriel Scherer
+
+- GPR#406: remove polymorphic comparison for Types.constructor_tag in compiler
+  (Dwight Guth,
+   review by Gabriel Radanne, Pierre Chambart, Mark Shinwell)
```

Comment:
I adjusted it to wrap only at 80 characters. If you want it all on one line even though it now would be longer than 80 characters on one line, let me know and I can do that as well.